Ports/dosfstools/package.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='dosfstools'
+version='4.2'
+useconfigure='true'
+use_fresh_config_sub='true'
+config_sub_paths=('config.sub')
+files="https://github.com/dosfstools/dosfstools/archive/refs/tags/v${version}.zip dosfstools-${version}.zip bed48396285facb98e1d323a523654977fb5ddf350597bf4b4de11ae667fd585"
```

Comment:
Why not the actual release tarball? That way we can skip relying on the user-installed `auto*` tools.